libs/unity/library/Runtime/Scripts/Media/WebcamSource.cs

```diff
@@ -212,6 +245,47 @@ protected async void OnEnable()
                 }
             }
 #endif
+#if PLATFORM_ANDROID
```

Comment:
nit: merge with previous `#if PLATFORM_ANDROID` section?

Reply:
This is the Android variant for the auto format block. I merged `#endif` + `#if` into `#elif` for clarity, but that should stay before the variable declaration. And the block above is about permissions. I prefer to have them clearly separated to show the steps. That's mostly a matter of taste though.

Reply:
No strong preferences. I guess putting the steps into separate methods would be the clearest way to show the steps though!